blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -5,6 +5,43 @@ common.date.readable.long=MMM DD YYYY h:mma Z
 common.date.readable.short=MMM DD h:mma Z
 common.pager.loading=Loading...
 common.pager.more=Show more
+# creation
```

Comment:
I have to reorganize the translation files as part of a refactor ticket that is coming up later this week, or early next week. I will update the description to reflect this change:  https://issues.jenkins-ci.org/browse/JENKINS-41139